doc/source/tutorials/algebraic_operations.ipynb

```diff
@@ -0,0 +1,287 @@
+    "    .timeseries()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you do not like the returned units, you can change that using the [rename()](https://pyam-iamc.readthedocs.io/en/stable/api/iamdataframe.html#pyam.IamDataFrame.rename) function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(\n",
+    "    df.divide(\"Primary Energy\", \"Population\", \"Energy / Capita\")\n",
+    "    .rename(unit={\"EJ / a / million\": \"EJ/yr/million\"})\n",
+    "    .timeseries()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or you can use the [convert_unit()](https://pyam-iamc.readthedocs.io/en/stable/api/iamdataframe.html#pyam.IamDataFrame.convert_unit) function;\n",
+    "see the [unit conversion tutorial notebook](https://pyam-iamc.readthedocs.io/en/stable/tutorials/unit_conversion.html) for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(\n",
+    "    df.divide(\"Primary Energy\", \"Population\", \"Energy / Capita\")\n",
+    "    .convert_unit(\"EJ / a / million\", \"GWh / yr\")\n",
+    "    .timeseries()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Overriding unit handling\n",
+    "\n",
+    "Even though **pint** is quite powerful, it does not always work as expected.\n",
+    "For example, *Mt CO2* is (strictly speaking) not a unit, but a species indicator *CO2* combined with a unit.\n",
+    "\n",
+    "For illustration, computing the emissions per capita will raise a [pint.UndefinedUnitError](https://pint.readthedocs.io/en/stable/developers_reference.html#pint.errors.UndefinedUnitError).\n",
+    "\n",
+    "We can override this behavior by setting ``ignore_units=True``; in this case, the unit of the returned timeseries data will be set to *unknown*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(\n",
+    "    df.divide(\"Primary Energy\", \"Population\", \"Emissions / Capita\",\n",
+    "              ignore_units=True)\n",
+    "    .timeseries()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also pass a string as the ``ignore_units`` keyword argument. Then, this string will be used as unit.\n",
+    "\n",
+    "Seeing that the unit of emissions is *Mt CO2* and Population is given in *million*, we know that the returned value should be given in *tons of CO2*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(\n",
+    "    df.divide(\"Primary Energy\", \"Population\", \"Emissions / Capita\",\n",
```

Comment:
same here. 
Text speaks about emissions, but here you take Primary Energy instead of Emissions|CO2